pkg/podimpersonation/podimpersonation.go

```diff
@@ -512,10 +512,12 @@ func (s *PodImpersonation) adminKubeConfig(user user.Info, role *rbacv1.ClusterR
 
 func (s *PodImpersonation) augmentPod(pod *v1.Pod, sa *v1.ServiceAccount, secret *v1.Secret, imageOverride string) *v1.Pod {
 	var (
-		zero = int64(0)
-		t    = true
-		f    = false
-		m    = int32(420)
+		zero      = int64(0)
+		t         = true
+		f         = false
+		m         = int32(420)
+		m2        = int32(384)
```

Comment:
Using Go's octal syntax makes it slightly more readable:
```suggestion
		m         = 0o644
		m2        = 0o600
```

Reply:
Done